src/pivot-table/data/extract-headers.ts

```diff
@@ -112,13 +118,24 @@ const extractHeaders = (
     );
   }
 
-  // Update canBeResized, isActivelySorted and colIdx on bottom row
+  // Update columnWidthLocation, isActivelySorted and colIdx on bottom row
   if (matrix.length) {
     for (const [colIdx, cell] of matrix[matrix.length - 1].entries()) {
       if (cell) {
-        cell.canBeResized = cell.isLeftDimension;
         cell.isActivelySorted = colIdx === (layoutService.layout.qHyperCube.activelySortedColumn?.colIdx ?? 0);
         cell.colIdx = cell.colIdx || colIdx;
+
+        if (cell.dimensionInfoIndex === PSEUDO_DIMENSION_INDEX) {
+          cell.columnWidthLocation = ColumnWidthLocation.Measures;
+        } else if (!cell.isLeftDimension) {
+          if (visibleLeftDimensionInfos.at(-1) === PSEUDO_DIMENSION_INDEX) {
+            cell.columnWidthLocation = ColumnWidthLocation.Measures;
+          } else if (visibleLeftDimensionInfos.indexOf(PSEUDO_DIMENSION_INDEX) !== -1) {
```

Comment:
Looks like this is what you are trying to evaluate?

```suggestion
          } else if (layoutService.hasPseudoDimOnLeft) {
```

Reply:
Yep